data/ecosystem/adopters.yaml

```diff
@@ -20,6 +20,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: Delivery Hero
+  url: https://www.deliveryhero.com/
+  components: [Collector, Go, Java]
+  reference: ''
```

Comment:
Would you have a reference like a talk or blog post we can link out to? This helps potential adopters to learn from your experience

Reply:
There is no blogpost. There was a talk at the OTel meetup in Berlin but AFAIK it was not recorded. Once there is any material available, I will link it here ;)